File: src/sdk-reference/kuzzle/code-example/constructor.java
try {
  Options options = new Options();
  options.autoResubscribe = false;

  Kuzzle kuzzle = new Kuzzle("kuzzle", options);

  System.out.println("Everything is ok");
} catch (KuzzleException e) {
  System.err.println(e.getMessage());
}

File: src/sdk-reference/kuzzle/code-example/constructor.cpp
options options = {0};
options.auto_resubscribe = false;

try {
  kuzzleio::Kuzzle kuzzle = new Kuzzle("kuzzle", &options);

  std::cout << "Everything is ok" << std::endl;
} catch (kuzzleio::KuzzleException e) {
  std:cerr << "Error: " << e.getMessage() << std::endl;
}
